ui/raidboss/data/03-hw/raid/a1s.js

```diff
@@ -69,8 +69,7 @@
     },
     {
       id: 'A1S Resin Bomb',
-      regex: Regexes.startsUsing({ id: 'E47', source: 'Oppressor' }),
-      condition: Conditions.targetIsYou(),
+      regex: Regexes.startsUsing({ id: 'E46', source: 'Oppressor', capture: false }),
```

Comment:
Should this be `['E46', 'E47']`? It looks like these are both resin bomb ids, or am I misunderstanding?

Reply:
Resin Bomb is a two-step cast, it's not a multiple-actors-doing-the-same-thing situation. You can see it in this [timestamped video](https://youtu.be/-yYS25mR7CI?t=19). You will see a long cast first, at the end of which four players are marked up with puddles. This first cast is `E46` and is the one that is important to stack for. The puddles activating is an invisible `E47` cast.